lab-jr/gulpfile.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('mocha');
+
+//use gulp to lint all the shits
+gulp.task('lint all the things', function(){
+  process.stdout.write('boops');//checking to see innards of function is printing to terminal
+  return gulp.src(['**/*.js', '!node_modules/**'])
+  .pipe(eslint());
```

Comment:
Go check out the gulp-eslint docs. There are a couple more pipes that you can chain on here to further your output and usability.